setup.py

```diff
@@ -35,7 +35,7 @@ def extensions():
     long_description = fh.read()
 
 s_required = ["cython>=0.17", "versioneer"]
-i_required = ["blosc", "contexttimer", "pyzfp"]
```

Comment:
you can use `extra_requires` to have multiple cases.
```
configuration = {...
        extras_require={'compression': ['blosc', 'zfp']}
```

And you have two installs:
`pip install pyrevolve` *no compression
`pip install pyrevolve[compression]` compression

Reply:
Can be for later if only want quick fix.